AK/Tests/TestTime.cpp

```diff
@@ -238,6 +238,11 @@ TEST_CASE(truncation)
     EXPECT_EQ(TIME(-2, -800'800'800).to_truncated_milliseconds(), -2'800);
     EXPECT_EQ(TIME(-2, -800'800'800).to_truncated_microseconds(), -2'800'800);
 
+    EXPECT_EQ(TIME(-9223372036854776, 193'000'000).to_truncated_milliseconds(), -0x7fff'ffff'ffff'ffff);
+    //EXPECT_EQ(TIME(-9223372036854776, 193'000'000).to_truncated_microseconds(), -0x7fff'ffff'ffff);
```

Comment:
@BenWiederhake not sure why this one doesn't pass, it's off by one. My brain needs a reboot ;)